frontend/src/entities/model/create/model-create.tsx

```diff
@@ -162,14 +162,14 @@ export function ModelCreate () {
                         navigate(`/project/${projectName}/model/${state.form.ModelName}`);
                     } else {
                         notificationService.generateNotification(
-                            `Failed to create model because: ${response.data}`,
+                            `Failed to create model with error: ${response.data}`,
```

Comment:
What are your thoughts about removing the `with error` from these types of messages? It seems redundant after saying it failed and adds extra text to what already could be a long message.

Reply:
I should probably just revert this. We use the `because` wording everywhere else